### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by giedriuzz**
Code without tests

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
+from main import TaskManager
```
Comment (Mindaugeliseth) [inline app.py:1]: Fix imports order: https://peps.python.org/pep-0008/#imports

File: app.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+from py_random_words import RandomWords
+import random
+from tqdm import tqdm
+import time
+import maskpass
+from datetime import timezone, timedelta
+import datetime
+from connect.connect import ConnectToRpi4
+
+# host = "localhost"
```
Comment (Mindaugeliseth) [inline app.py:11]: No commented code in a final version unless it is necessary

File: app.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+#     host=host,
+#     port=port,
+#     user_name=user_name,
+#     user_passwd=password,
+#     db_name=db_name,
+#     collection_name=collection_name,
+# )
+db = ConnectToRpi4()
+
+dictionary = {}
```
Comment (Mindaugeliseth) [inline app.py:30]: what dictionary?

File: app.py [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+while True:
+    field_name = input("Name of field: ")
```
Comment (Mindaugeliseth) [inline app.py:33]: Are you guarding on any wrong input types with all `input` statements?

File: app.py [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+        year_diff = int(input("Enter year difference: "))
+        dictionary.update({field_name: ("utc", year_diff)})
+    elif field_type == 5:
+        print("*** Field created successfully! ***\n")
+        break
+
+i = int(input("How many documents do you want to create? "))
+
+
+for _ in tqdm(range(0, i), desc="Progress..."):
```
Comment (Mindaugeliseth) [inline app.py:59]: tqdm?

File: app.py [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+    document = {}
```
Comment (Mindaugeliseth) [inline app.py:60]: what document?

File: app.py [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+            random_hours = random.randint(1, 59)
+            random_minutes = random.randint(1, 59)
+            random_seconds = random.randint(1, 59)
+            new_date = dt - timedelta(
+                days=random_years,
+                hours=random_hours,
+                minutes=random_minutes,
+                seconds=random_seconds,
+            )
+            # only_date = new_date.strftime("%Y-%m-%d") # string format if need by task
```
Comment (Mindaugeliseth) [inline app.py:84]: Move to outer re-usable free function

File: connect/connect.py [hunk 8]
```diff
@@ -0,0 +1,26 @@
+from pymongo import MongoClient
```
Comment (Mindaugeliseth) [inline connect/connect.py:1]: Import order, the same as in previous module

**Review by Mindaugeliseth — APPROVED** [review]

**Review by Mindaugeliseth — APPROVED** [review]

File: app.py (hunk 5 above)
Comment (giedriuzz) [reply app.py:59]: > tqdm?

yes its for visualization of creating a document in a database

File: app.py (hunk 6 above)
Comment (giedriuzz) [reply app.py:60]: > what document?

changed to "document_of_collection"

File: app.py (hunk 3 above)
Comment (giedriuzz) [reply app.py:30]: changed to "dict_of_collection"

**Comment by giedriuzz** [comment]
project postponed

**Pull request merged**